core/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/ActionManagerImpl.java

```diff
@@ -105,8 +106,10 @@ private void registerDefaultActionGroups() {
         DefaultActionGroup mainContextMenuGroup = new DefaultActionGroup(IdeActions.GROUP_MAIN_CONTEXT_MENU, false, this);
         registerAction(IdeActions.GROUP_MAIN_CONTEXT_MENU, mainContextMenuGroup);
 
-        DefaultActionGroup buildContextMenuGroup = new DefaultActionGroup(IdeActions.GROUP_BUILD_CONTEXT_MENU, false, this);
+        DefaultActionGroup buildContextMenuGroup = new DefaultActionGroup("Maven", true, this);
```

Comment:
I think that constant for group's name isn't necessary. Because it used in one place only.